source/configure/configuration-settings.rst

```diff
@@ -6615,6 +6615,19 @@ Defines the threshold in hours beyond which older completed database jobs are re
 | This feature's ``config.json`` setting is ``"JobSettings.CleanupJobsThresholdDays": -1`` with numerical input.     |
 +--------------------------------------------------------------------------------------------------------------------+
 
+Clean Up Outdated Database Entries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+|all-plans| |self-hosted|
+
+This setting isn't available in the System Console and can only be set in ``config.json``.
```

Comment:
```suggestion
This setting only applies to configuration on database. And it isn't available in the System Console hence it can be set via mmctl or changing configurations itself.
```

Reply:
Sorry for the late reply @cwarnermm 🙏 I was off last week and this one escaped from me.. I meant the configuration needs to be changed directly from the database. Consider that this setting has no meaning for the file based configurations hence we can not ask them to update the file :)

Reply:
Thanks, @isacikgoz! I've updated the text based on your input. This PR should be ready for your review.